plugins/core/gov.nasa.jpl.imce.oml.viewpoint/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: gov.nasa.jpl.imce.oml.viewpoint;singleton:=true
+Bundle-Version: 1.0.0.qualifier

```

Comment:
For version management (see https://github.com/JPL-IMCE/gov.nasa.jpl.imce.oml.tycho#to-set-versions-everywhere), set the bundle version to: 
0.7.0.qualifier